packages/vue/src/components/organisms/SfMegaMenu/SfMegaMenu.stories.scss

```diff
@@ -1,5 +1,5 @@
 @import "~@storefront-ui/shared/styles/helpers";
-.sb-mega-menu {
+.sf-mega-menu {
```

Comment:
this class name has a different prefix and it's not a mistake :slightly_smiling_face: